### PR TITLE
feat(editor): Update panning and selection keybindings on new canvas

### LINKED ISSUE #11519: New Alpha Editor Zoom In and Pan/Drag Bug
### Bug Description

By pressing + button, the Editor **Zoom Out** which should **Zoom In** as the use case.
Second thing is by Pressing CMD and we use mouse it doesn't active the Pan/Drag feature.

### To Reproduce

1. Go to Workflow Editor.
2. Switch to New Canvas Editor.
3. Now try above shortcuts i mentioned in Bug Description. 

### Expected behavior

By pressing + button it should Zoom In.
By pressing CMD + mouse click, it should drag or move the canvas.

### Operating System

Docker

### n8n Version

1.65.2

### Node.js Version

18.16

### Database

SQLite (default)

### Execution mode

main (default)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

Fixes #11519

Latest VueFlow [update](https://vueflow.dev/changelog/#_1-41-3) fixes some bugs, allowing us to remove workarounds for the following behaviours:

- Use Space, Control or Meta keys to activate panning
- Use Middle mouse button to pan natively

https://github.com/user-attachments/assets/6ea345bf-6ebd-4222-ae49-78a4bbfc9d31

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/CAT-278/support-meta-left-mouse-click-to-navigate-the-canvas

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/canvas/Canvas.vue [hunk 1]
```diff
@@ -143,15 +133,16 @@ const disableKeyBindings = computed(() => !props.keyBindings);
...
-const panningKeyCode = ' ';
-const isPanningEnabled = ref(false);
+const isPanning = ref(false);
+const panningKeyCode = ref<string[]>([' ', controlKeyCode]);
+const panningMouseButton = ref<number[]>([1]);
 const selectionKeyCode = ref<true | null>(true);
 
-onKeyDown(panningKeyCode, () => {
+onKeyDown(panningKeyCode.value, () => {
 	setPanningEnabled(true);
```
Comment (r00gm) [inline packages/editor-ui/src/components/canvas/Canvas.vue:142]: @alexgrozav do we still need this `setPanningEnabled`?

Comment (alexgrozav) [reply packages/editor-ui/src/components/canvas/Canvas.vue]: Nope, good point. Removed it.

**Review by r00gm — APPROVED** [review]

**Comment by alexgrozav** [comment]
Fixes #11519

**Comment by janober** [comment]
Got released with [`n8n@1.67.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.67.0)

**Pull request merged**